src/components/Connection/index.tsx

```diff
@@ -3,8 +3,31 @@ import { ConnectionModalProps } from './type';
 import * as S from './style';
 import { Svg } from '@/components/Svg';
 import { mainThemeColor } from '@/styles/theme';
+import { useRef, useState } from 'react';
+import { ConnectionQueries } from '@/apis/connection';
+import { EachUserInfo } from '@/apis/connection/type';
+import { SearchErrorHandler } from '@/apis/connection/error';
+import { toast } from 'react-toastify';
+import UserInfo from './UserInfo';
 
 export default function ConnectionModal({ isOpen }: ConnectionModalProps) {
+  const emailRef = useRef<HTMLInputElement>(null);
+  const mutationSearch = ConnectionQueries.SearchQuery();
+  const [userList, setUserList] = useState<Array<EachUserInfo>>([]);
```

Comment:
```suggestion
  const [userList, setUserList] = useState<EachUserInfo[]>([]);
```
이렇게도 사용할 수 있는데, `Array`라는 문구를 쓰는게 오히려 배열을 확인하기 좋아서 좋아보일 수 있는데, 간소화 시켜서 제가 제안드리는 방식도 좋아보입니다. 

이 부분은 컨벤션으로 정해야 할 것 같은데, 어떠신가요?

Reply:
저도 여기서만 저렇게 쓴거같아서, 제안 주신 방법으로 하죠 저도